data/json/items/resources/metals.json

```diff
@@ -138,5 +138,21 @@
     "color": "light_gray",
     "ammo_type": "components",
     "qualities": [ [ "HAMMER", 1 ] ]
+  },
+  {
+    "type":"GENERIC",
```

Comment:
Not yet linted but the file is close to perfect so try and avoid inconsistent ordering here